packages/grid/_modules_/grid/models/api/gridFilterApi.ts

```diff
@@ -39,7 +33,7 @@ export interface GridFilterApi {
    * Changes the [[GridLinkOperator]] used to connect the filters.
    * @param {GridLinkOperator} operator The new link operator. It can be: `"and"` or `"or`".
    */
-  applyFilterLinkOperator: (operator: GridLinkOperator) => void;
+  setFilterLinkOperator: (operator: GridLinkOperator) => void;
```

Comment:
This makes the PR a breaking change. I've updated the description to provide an alternative.

Reply:
Is this change necessary? The logic of the method is more or less the same.